src/eva/vision/data/wsi/backends/openslide.py

```diff
@@ -32,8 +32,28 @@ def level_downsamples(self) -> Sequence[float]:
     @override
     def mpp(self) -> float:
         # TODO: add overwrite_mpp class attribute to allow setting a default value
-        x_mpp = float(self._wsi.properties["openslide.mpp-x"])
-        y_mpp = float(self._wsi.properties["openslide.mpp-y"])
+        if self._wsi.properties.get("openslide.mpp-x") and self._wsi.properties.get(
+            "openslide.mpp-y"
+        ):
+            x_mpp = float(self._wsi.properties["openslide.mpp-x"])
+            y_mpp = float(self._wsi.properties["openslide.mpp-y"])
+
+        elif (
+            self._wsi.properties.get("tiff.XResolution")
```

Comment:
Seems that this is tiff specific. Can you maybe test what happens for other formats such as .svs and mirax?

Reply:
yes for svs this key is missing, instead there is the additional property `aperio.MPP`. 

Covering every file format for openslide 3.4 might not be feasible, however, we could limit this to support the formats of the tasks we implement?

Reply:
sure, let's do that!